front/pages/api/login.ts

```diff
@@ -246,6 +251,14 @@ async function handler(
         });
       }
 
+      if (EMAILS_TO_AUTO_UPGRADE.includes(u.email)) {
```

Comment:
Can we move it to the place the workspace is created? A bit worry this will run at each login for that user?